src/Default/default.ts

```diff
@@ -191,7 +191,7 @@ export class DefaultContainer extends WebContainerBase {
         return ObjectTransform.transformProperties(options, this.windowOptionsMap);
     }
 
-    protected wrapWindow(containerWindow: any) {
+    public wrapWindow(containerWindow: any) {
```

Comment:
wrapWindow should have been public on all containers.  Changed now for consistency since I had to make it public in ElectronContainer since it is invoked from within ElectronContainerWindow now from getGroup().  Ideally it would be internal but that doesn't exist in TypesScript right now.

Reply:
Can we not get away with doing something like https://github.com/Microsoft/TypeScript/issues/5228#issuecomment-344132670 and keep the interface clean?

Reply:
It might but I would have to nest ElectronContainerWindow within ElectronContainer.  I want it to be public anyway so someone can wrap a window if necessary (this might prove useful for our internal container especially).